### PR TITLE
MFTF: Customer Subscribes To Newsletter Subscription On StoreFront

### PR DESCRIPTION
**Pull request by DmitryTsymbal**
This PR contains MFTF Test for customer's newsletter subscription.

**Steps to reproduce:**
1 - Create a customer
2 - Login on storefront as created customer
3 - Navigate to newsletter page
4 - Subscribe to newsletter
5 - Assert success message

### CODE REVIEW

File: app/code/Magento/Customer/Test/Mftf/Section/StorefrontCustomerLoginFormSection.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<sections xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:noNamespaceSchemaLocation="urn:magento:mftf:Page/etc/SectionObject.xsd">
+    <section name="StorefrontCustomerLoginFormSection">
```
Comment (lbajsarowicz) [inline app/code/Magento/Customer/Test/Mftf/Section/StorefrontCustomerLoginFormSection.xml]: Why don't you use `StorefrontCustomerSignInFormSection`?

File: app/code/Magento/Customer/Test/Mftf/ActionGroup/StorefrontCustomerLoginActionGroup.xml [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+<!--
+ /**
+  * Copyright © Magento, Inc. All rights reserved.
+  * See COPYING.txt for license details.
+  */
+-->
+
+<actionGroups xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+              xsi:noNamespaceSchemaLocation="urn:magento:mftf:Test/etc/actionGroupSchema.xsd">
+    <actionGroup name="StorefrontCustomerLoginActionGroup">
```
Comment (lbajsarowicz) [inline app/code/Magento/Customer/Test/Mftf/ActionGroup/StorefrontCustomerLoginActionGroup.xml]: I like the fact that you want to introduce proper naming for actions, but if you want to do so - deprecate old one: `LoginToStorefrontActionGroup`.

**Review by lbajsarowicz — CHANGES_REQUESTED** [review]
:hand:  Please review the comments.

**Comment by DmitryTsymbal** [comment]
Hello @lbajsarowicz , I have re-used existing Action Groups. Could you check it please?
Thanks

**Review by lbajsarowicz — APPROVED** [review]
:heavy_check_mark: Thank you for your contribution

**Comment by magento-engcom-team** [comment]
Hi @lbajsarowicz, thank you for the review.
[ENGCOM-7148](https://jira.corp.magento.com/browse/ENGCOM-7148) has been created to process this Pull Request
:eight_spoked_asterisk: @lbajsarowicz, could you please add one of the following labels to the Pull Request?

| Label | Description |
|--------------------------------|-----------------------------------------------------------------|
| ```Auto-Tests: Covered``` | All changes in Pull Request is covered by auto-tests |
| ```Auto-Tests: Not Covered``` | Changes in Pull Request requires coverage by auto-tests |
| ```Auto-Tests: Not Required``` | Changes in Pull Request does not require coverage by auto-tests |

**Comment by engcom-Alfa** [comment]
:heavy_check_mark: QA Passed

**Comment by engcom-Echo** [comment]
@magento run all tests

**Comment by engcom-Echo** [comment]
Failed functional tests not related to the changes in this PR

**Pull request merged**